Route/Quote/Invoice.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace monsieurgourmand\Bundle\InterfaceBundle\Route\Quote;
+
+use monsieurgourmand\Bundle\InterfaceBundle\Service\MGD;
+use monsieurgourmand\Bundle\InterfaceBundle\Model\Quote;
+use monsieurgourmand\Bundle\InterfaceBundle\Model\Invoice as InvoiceModel;
+
+class Invoice
+{
+    private $entity;
+    private $url;
+    private $master;
+
+    public function __construct(Quote $quote)
+    {
+        $this->master = $quote->getMaster();
+        $this->entity = InvoiceModel::class;
+        $this->url = "/events/" . $quote->getEvent()->getIdEvent() . "/quotes/" . $quote->getId() . '/invoices';
+    }
+
+    public function getAll(string $format = MGD::FORMAT_OBJECT)
+    {
+        $params = array();
```

Comment:
```suggestion
        $params = [];
```
Pourquoi il n'est pas dans les arguments de la fonction comme d'autres getAll ?